src/main/java/domain/piece/Pawn.java

```diff
@@ -0,0 +1,98 @@
+package domain.piece;
+
+import domain.position.Position;
+import domain.position.Rank;
+import java.util.Map;
+
+public class Pawn extends AbstractPiece {
+    private static final Map<Color, Rank> INITIAL_RANK = Map.of(Color.WHITE, Rank.TWO, Color.BLACK, Rank.SEVEN);
```

Comment:
```suggestion
    private static final Map<Color, Rank> INITIAL_RANK = Map.of(
        Color.WHITE, Rank.TWO,
        Color.BLACK, Rank.SEVEN
    );
```
여기도 줄바꿈을 이용해서 가독성을 높여볼 수 있을 것 같아요.
Pawn 쪽에 InitalRank 를 명시해줘서 찾기 정말 좋네요 👍